spec/javascripts/app/acuant/document_capture_fallback_spec.js

```diff
@@ -1,82 +0,0 @@
-import sinon from 'sinon';
```

Comment:
I think we'll also want to remove the file `spec/javascripts/support/acuant/document_capture_dom.js` ?

Reply:
Yep, done